.travis.yml

```diff
@@ -19,7 +19,10 @@ before_install:
   - sudo make base
   - sudo make apt-openfoam-deps apt-simphony-deps apt-lammps-deps apt-mayavi-deps apt-aviz-deps
   - make fix-pip
-  - pip install --allow-external PIL --allow-unverified PIL PIL
+  - sudo pip uninstall -y argparse
```

Comment:
For the tracebacks it looks that the new numpy has a problem with argprase. You might want to do `sudo pip install --upgrade argparse` in `fix-pip` or  `sudo apt-get remove python-argparse` in `apt-symphony`.